client/src/pages/fundraisers/FundraiserCreate.js

```diff
@@ -53,14 +53,12 @@ class FundraiserCreate extends Component {
         description: "",
         amount: "",
         deadline: dayjs().add(30, "days"),
-        live: false,
+        live: false
```

Comment:
This gets it working but remove it in the future as fundraiser create should not be playing with this property in the frontend.

Reply:
Noted, i believe this addition was from another contribution